### PR TITLE
[PR1] DEV-68 Refactored ReceiveTable

### PR DESCRIPTION
**Pull request by smorsua**
This PR introduces a major refactor to the ReceiveTable Component. To be able to handle the load of packets, I've used a library called `react-window`. When you have a very long list but only some elements are visible, you can use this library to only create the visible components, meaning that all **of the unseen items are not created**, therefore not updated when nwe pakcet updates arrive, thus increasing performance.
[screen-capture (7).webm](https://user-images.githubusercontent.com/114338434/206741083-9b63497b-9ce1-48f8-a4c6-80851d8b0894.webm)

The `VariableSizeList` component provided by the `react-window` library is used in `BoardSection`: 

image

I've also used `AutoSizer`, which is an additional component that's needed so the list is 100% width of the parent. Both react-window and autosizer are extracts from a bigger library called react-virtualized, which provides a multitude of components that allow you to increase performance in the way I described earlier. I you are interested in how react-window works, this might help: 
image

Basically, with the parameter of total size of the list, size per item, and position of scroll, the `VariableSizeList` is able to determine wich components should be rendered and where. Then it uses absolute positioning to place each item in the correct position. Everytime the scroll event is fired, all this calculation are done again.

Other major change is that **instead of using the HTML table element, I've used flex and grid layout**. This simplifies the component and makes it compatible with `react-window`. If you don't understand this last reason, ask me if interested.

### CODE REVIEW

**Review by jmaralo — APPROVED** [review]

**Review by lipezaballa — APPROVED** [review]

File: src/components/ReceiveTable/BoardSection/BoardSection.tsx [hunk 1]
```diff
@@ -1,24 +1,67 @@
...
-          {board.name}
-        </td>
-      </tr>
-      {Object.values(board.packets).map((packet) => {
-        return <PacketRow key={packet.id} packet={packet}></PacketRow>;
-      })}
-    </>
+    <div className={styles.wrapper}>
+      <div className={styles.header}>
+        <Caret
```
[the remark this answers is not in the document]
Comment (smorsua) [reply src/components/ReceiveTable/BoardSection/BoardSection.tsx:20]: It's always visible
